public/js/main.js

```diff
@@ -7,6 +7,7 @@ require('o-header-services');
 require('./filterbar');
 require('./click-helper');
 require('./demos');
+require('./sticky-sidebar');
```

Comment:
We seem to be using both `import` and `require`, is it possible to settle on one, ideally `import` as it is the method which is going to be supported by browsers in the future